tests/auth_basic_test.py

```diff
@@ -64,41 +65,63 @@ def test_signup_route(client, capture_templates, setup_logger):
         raise
 
 
-def test_signup_post_invalid_email(client, capture_templates, setup_logger):
+param_list = [
+    ("bad_email", "TestP@ssword!", HTTPStatus.OK),  # bad email
+    ("test@example.com", "wrongpassword", HTTPStatus.OK),  # wrong password
+    # SQL injections
+    ("test@example.com' OR '1'='1' --", "TestP@ssword!", HTTPStatus.OK),
+    (
+        "test@example.com",
+        "wrongpassword' UNION SELECT 1, username, password FROM users --",
+        HTTPStatus.OK,
+    ),
+]
+
+
+@pytest.mark.parametrize("email,password,http_status", param_list)
+def test_signup_post_invalid_email(
+    client, capture_templates, setup_logger, email, password, http_status
```

Comment:
This test should have an `email`, `password`, and `confirm_password` parameter.

Reply:
updated!